docs/validators/sigstore_cosign.md

```diff
@@ -92,13 +92,13 @@ kubectl run altsigned --image=docker.io/securesystemsengineering/testimage:co-si
 | `name` | - | :heavy_check_mark: | See [basics](../basics.md#validators). |
 | `type` | - | :heavy_check_mark: | `cosign`; the validator type must be set to `cosign`. |
 | `trustRoots[*].name` | - | :heavy_check_mark: | See [basics](../basics.md#validators). |
-| `trustRoots[*].key` | - | :heavy_check_mark: if not using keyless | See [basics](../basics.md#validators). ECDSA/RSA public key from `cosign.pub` file or [KMS URI](https://github.com/sigstore/cosign/blob/main/KMS.md). See additional notes [below](#kms-support). |
-| `trustRoots[*].keyless.Issuer` | - | :heavy_check_mark: if not using a key | Name of the OIDC provider which attests the identity. This is a regex, so either the full url of the OIDC provider can be used or only parts of it. |
-| `trustRoots[*].keyless.subject` | - | :heavy_check_mark: if not using a key | Name of the identity that created the keyless signature. Usually an email address. This is a regex, so either the full email address can be used or only parts of it (e.g. `your.mail@example.org` or `your.mail`). |
+| `trustRoots[*].key` | - | :heavy_check_mark: if not using keyless | See [basics](../basics.md#validators). ECDSA/RSA public key from `cosign.pub` file or [KMS URI](https://docs.sigstore.dev/key_management/overview/#basic-usage). See additional notes [below](#kms-support). |
+| `trustRoots[*].keyless.Issuer` | - | :heavy_check_mark: if not using a key | Regex for the OIDC provider name which attests the identity. So, either the full URL of the OIDC provider can be used or only parts of it. |
+| `trustRoots[*].keyless.subject` | - | :heavy_check_mark: if not using a key | Regex of the identity that created the keyless signature. Usually an email address. So, either the full email address can be used or only parts of it (e.g. `your.mail@example.org` or `your.mail`). |
```

Comment:
This would be the docs for *Regex, while the docs for Issue and subject are missing